exonum_launcher/launch_state.py

```diff
@@ -11,7 +11,7 @@ class LaunchState:
     def __init__(self) -> None:
         self._pending_configs: Dict[Configuration, List[str]] = dict()
         self._pending_deployments: Dict[Artifact, List[str]] = dict()
-        self._pending_migrations: Dict[Tuple[str, Artifact], List[str]] = dict()
+        self._pending_migrations: Dict[Tuple[str, Artifact, int], List[str]] = dict()
```

Comment:
IMHO it makes sense to use a class instead of tuple.

Reply:
Yeah. I've thought about this as well. I think to do this and more related to it in scope of other PR. It requires a massive refactoring and extending test coverage, I suppose. But a lack of time, as you know.

Reply:
Sure, no problem (: